dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -93,22 +93,45 @@ done
 # Set configuration common to both minishift and openshift
 # --------------------------------------------------------
 
-CHE_MULTI_USER=${CHE_MULTI_USER:-"false"}
 DEFAULT_COMMAND="deploy"
 COMMAND=${COMMAND:-${DEFAULT_COMMAND}}
 
+export CHE_EPHEMERAL=${CHE_EPHEMERAL:-false}
+CHE_USE_ACME_CERTIFICATE=${CHE_USE_ACME_CERTIFICATE:-false}
+
+CHE_FABRIC8_MULTITENANT=${CHE_FABRIC8_MULTITENANT:-false}
+CHE_FABRIC8_USER__SERVICE_ENDPOINT=${CHE_FABRIC8_USER__SERVICE_ENDPOINT:-"https://api.openshift.io/api/user/services"}
+CHE_FABRIC8_WORKSPACES_ROUTING__SUFFIX=${CHE_FABRIC8_WORKSPACES_ROUTING__SUFFIX:-"8a09.starter-us-east-2.openshiftapps.com"}
+if [ "${CHE_FABRIC8_MULTITENANT}" == "true" ]; then
+  CHE_DOCKER_ENABLE__CONTAINER__STOP__DETECTOR=false
+  DEFAULT_CHE_MULTI_USER=true
+else
+  CHE_DOCKER_ENABLE__CONTAINER__STOP__DETECTOR=true
+  DEFAULT_CHE_MULTI_USER=false
+fi
+
+CHE_MULTI_USER=${CHE_MULTI_USER:-${DEFAULT_CHE_MULTI_USER}}
+
 if [ "${CHE_MULTI_USER}" == "true" ]; then
   DEFAULT_CHE_KEYCLOAK_DISABLED="false"
-  CHE_DEDICATED_KEYCLOAK=${CHE_DEDICATED_KEYCLOAK:-"true"}
-  DEFAULT_CHE_IMAGE_REPO="docker.io/eclipse/che-server-multiuser"
+  
+  if [ "${CHE_FABRIC8_MULTITENANT}" == "true" ]; then
+    CHE_DEDICATED_KEYCLOAK="false"
+    DEFAULT_CHE_IMAGE_REPO="docker.io/rhchestage/che-server"
```

Comment:
That should be set to: `DEFAULT_CHE_IMAGE_REPO="docker.io/rhchestage/che-server-multiuser"`

Reply:
Fixed